src/plugins/newsfeed/public/plugin.tsx

```diff
@@ -27,8 +27,10 @@ import { FetchResult, NewsfeedPluginInjectedConfig } from '../types';
 import { NewsfeedNavButton, NewsfeedApiFetchResult } from './components/newsfeed_header_nav_button';
 import { getApi } from './lib/api';
 
-export type Setup = void;
-export type Start = void;
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface Setup {}
```

Comment:
Maybe `export type Setup = object;` ? Then you would not need eslint-disable

Reply:
Addressed in 9749dc0!